_service-details/secchamps.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Managed Security Champions Programme
+subtitle: Helping you build and improve this
+layout: service
+hero_height: is-small
+backlink: /services
+backname: Value
+---
+
+The industry is pretty unanimous that your organization needs a security champions programme. But where are you going to find time to develop a high value and high impact programme for this, alongside your already full schedule?

```

Comment:
I think this needs better purpose. "Everyone else is doing it" or "everyone thinks you should do it" is not very powerful. Instead perhaps the benefits of a champ[ions programme, or even the pain that comes without it? (2nd sentence I think is good)

Reply:
Ok, I tried to make it a little more punchy.

Reply:
I like this!